rebasehelper/helpers/macro_helper.py

```diff
@@ -38,6 +38,8 @@ class MacroHelper(object):
         '_sysconfdir',
         '_bindir',
         '_libdir',
+        'python2_sitelib',
+        'python3_sitelib',
```

Comment:
Is there a reason to add these macros to this position? I think it would be nicer if the whole list was sorted alphabetically.

Reply:
I don't think there is the reason for the current list ordering. I put python{2,3}_sitelib after _libdir because they are related. But you are right that it may be better to sort alphabetically. Do you suggest taking the starting underscores into consideration or should I ignore them?

Reply:
No, please, don't ignore the underscores, that would be confusing (to me at least).